examples/swapAndBridge.ts

```diff
@@ -118,9 +119,20 @@ const swapAndBridge: Parameters<typeof withAnvilProvider>[0] = async (
     userAddr,
     encodedSignature
   );
+
+  let gasLimit: string;
+  try {
+    gasLimit = (await provider.estimateGas({
+      to: factory,
+      data: hooksCalldata
+    })).toString();
+  } catch (err) {
+    gasLimit = 100_000_000n.toString()
```

Comment:
I'm not sure about this, given that there are block limits of 30M on mainnet, or 17M on gnosis. It would work on arbitrum though :laughing: This may cause issues for simulations in the backend if a user tries to set such a large gas limit. I'd default to something like `block limit / 2`

Reply:
outdated, this doesnt exist anymore